### PR TITLE
Poster halls

### PR DESCRIPTION
**Pull request by colinhowe**
Basic implementation of poster halls. As my time remaining is short this PR focuses on getting the logic right and then it should be easy for someone to come across and add missing functionality (tags, author name etc)

This PR also removes screening rooms as they're unused and depended on a lot of the old poster logic.

image

image

### CODE REVIEW

**Comment by colinhowe** [comment]
Closes https://github.com/sparkletown/internal-sparkle-issues/issues/1852

File: src/components/admin/PosterConfiguration/PosterConfiguration.tsx [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+  const [
+    { loading: isUpdating, error: updateError, value: result },
+    updatePosters,
+  ] = useAsyncFn(async (data) => {
+    const csvFiles = data.csvFile;
+    if (!csvFiles.length) {
+      throw new Error("Must provide a file to upload");
+    }
+    const fileContents = await new Promise<string>((resolve, reject) => {
```
Comment (alexxpetrov) [inline src/components/admin/PosterConfiguration/PosterConfiguration.tsx:64]: I think we'd benefit if this parsing logic was separated from the actual render
For now, can we please add `@debt` comments, so we can clean it up once we have time for it?

File: src/components/admin/PosterConfiguration/PosterConfiguration.tsx [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+              THUMBNAIL: "thumbnailUrl",
+              DESCRIPTION: "description",
+              "EMBED URL": "embedUrl",
+              "POSTER ID": "posterId",
+            }[String(column).toUpperCase()])
+        ),
+      skip_empty_lines: true,
+    });
+
+    posterRecords.forEach((posterRecord, idx) => {
```
Comment (alexxpetrov) [inline src/components/admin/PosterConfiguration/PosterConfiguration.tsx:91]: Let's add `@debt` here

**Review by alexxpetrov — APPROVED** [review]
LGTM
Let's add `@debt` comments for future improvements, merge conflicts and we'll be good to go

**Pull request merged**